fieldobjectpermissions/apps.py

```diff
@@ -133,6 +132,7 @@ class FieldObjectPermissionsConfig(AppConfig):
     verbose_name = 'Field Object Permissions'
 
     def ready(self):
+        # from .backends import GROUP_FIELDS, OWNER_FIELDS, GROUP_ACTIONS, OWNER_ACTIONS
```

Comment:
Can you remove this comment? Looks like it was left over from a different approach.